examples/monaco-graphql-react-vite/src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import { ReactElement, useEffect, useRef, useState } from 'react';
 import { getIntrospectionQuery, IntrospectionQuery } from 'graphql';
 import { Uri, editor, KeyMod, KeyCode, languages } from 'monaco-editor';
-import { initializeMode } from 'monaco-graphql/src/initializeMode';
+import { initializeMode } from 'monaco-graphql/dist/initializeMode';
```

Comment:
@acao this is big mistake 🥲

Reply:
i couldn't get the dist bundling working before, but this example is from last year, it seems monaco graphql bundling had changed since